### DISCUSSION TITLE
possibility to change rtsp-simple-server.yml path

### DISCUSSION THREAD

**Discussion by Emilgardis** [bluenviron/mediamtx#906] [Questions] [answered]
Hi! Is it possible to configure (e.g via env variable or cmd line) the location of the config file?

**Comment by aler9** [comment, marked as answer]
Yes, you can do it by passing the path as the first argument of the executable:

```
./rtsp-simple-server /path/to/config.yml
```

or

```
docker run --rm -it --network=host aler9/rtsp-simple-server /path/to/config.yml
```

**Reply by Emilgardis** [reply]
great, I couldn't find it documented and tried parsing through the code but my Go abilities are pretty lacking. Thank you!

[a comment by an automated account is not shown]
